rtv/content.py

```diff
@@ -510,3 +574,52 @@ def get(self, index, n_cols=70):
         data['offset'] = 0
 
         return data
+
+
+class MultiredditContent(Content):
+
+    def __init__(self, multireddits, loader):
+
+        self.name = "Multireddits"
+        self.order = None
+        self._loader = loader
+        self._multireddits = multireddits
+        self._multireddit_data = []
+
+        try:
+            self.get(0)
+        except IndexError:
+            raise exceptions.SubscriptionError('No multireddits')
+
+    @classmethod
+    def from_user(cls, reddit, multireddits, loader):
+        multireddits = (m for m in multireddits)
```

Comment:
Is there a reason that you didn't follow the SubscriptionContent() convention and do something like this here?

```python
multireddits = reddit.get_my_multireddits(limit=None)
```